telemetry/specs/INT.mdk

```diff
@@ -377,6 +377,44 @@ Considering all these factors, the INT specification requires that an INT
 switch **must not fragment** packets in order to append INT information to
 the packet.
 
+### Checksum Update
+
+INT may be transported over an L4 protocol such as TCP or UDP, or over an
+encapsulation header that includes an L4 header, such as VXLAN. In these cases,
+an INT source or transit switch may be required to update the L4 Checksum of
+the encapsulating header.
+
+In some cases, an L4 Checksum update is not necessary. For example, when UDP is 
+transported over IPv4, it is possible to assign a zero Checksum, causing the 
+receiver to ignore the value of the Checksum field. For UDP over IPv6, there
+are specific use cases in which it is possible to assign a zero Checksum 
+(as defined in RFC 6936).
+
+If an L4 Checksum update is required, an INT source/transit switch may perform
+it in one of two ways:
+
+* Update the L4 Checksum field such that the new value is equal to the Checksum 
+of the new packet, after the INT-related updates, or
+
+* Assign a value to the Checksum Complement metadata which guarantees that the 
+existing L4 Checksum is the correct value of the packet after the INT-related 
+updates.
+
+The motivation for the Checksum Complement is that some hardware implementations
+process data packets in a serial order, which may impose a problem when INT
+fields and metadata that reside after the L4 Checksum field are inserted or 
+modified. Therefore, the Checksum Complement metadata, if present, is the last
+metadata field in the stack. 
+
+Note that when the Checksum Complement metadata is present source/transit 
+switches may choose to update the L4 Checksum field instead of using the
+Checksum Complement metadata. In this case the Checksum Complement will be 
+assigned zero. A host that verifies the L4 Checksum will be unaffected by 
```

Comment:
In general, this spec uses 0xFFFFFFFF as the reserved value for metadata corresponding to an instruction that an INT hop does not process. Using a consistent reserved value would simplify implementations. An implementation can simply insert this single reserved value for any bits in the instruction bit mask that it does not support, without understanding and distinguishing between the bits that it does not support.